src/autoload/CFG_GLPI.php

```diff
@@ -650,3 +650,4 @@
 
 $CFG_GLPI['process_types'] = ['Computer'];
 $CFG_GLPI['environment_types'] = ['Computer'];
+$CFG_GLPI['default_illustration'] = 'request-service';
```

Comment:
I think it is overkill to put this in a CFG_GLPI entry. You could use a constant instead, in the IllustrationManager class for instance.